data/json/items/book/cooking.json

```diff
@@ -441,5 +441,62 @@
     "intelligence": 12,
     "time": "35 m",
     "fun": -1
+  },
+  {
+    "id": "cookbook_liverforkids",
+    "type": "BOOK",
+    "name": { "str": "Liver-Licious Recipes Your Kids Will Love", "str_pl": "copies of Liver-Licious Recipes" },
+    "description": "This cookbook is written by liver enthusiast Tamsyn Beckerleg.  Inside you'll find a great deal of anecdotes about the near-certain ruination of children who refuse to eat their liver.",
+    "weight": "600 g",
+    "volume": "800 ml",
+    "price": 1999,
+    "price_postapoc": 3000,
+    "bashing": 2,
+    "material": [ "paper" ],
+    "symbol": "?",
+    "color": "blue",
+    "skill": "cooking",
+    "required_level": 1,
+    "max_level": 2,
+    "intelligence": 5,
+    "time": "45 m"
+  },
+  {
+    "id": "cookbook_daintydishes",
+    "type": "BOOK",
+    "name": { "str": "Dainty Dishes Fit for a King", "str_pl": "copies of Dainty Dishes" },
+    "description": "This cookbook contains many fanciful recipes. Some of them are even worth the trouble to actually prepare. Lovely pen and ink illustrations of overweight knights and gluttonous monks line the pages.",
+    "volume": "900 ml",
+    "price": 3000,
+    "price_postapoc": 4700,
+    "max_level": 3,
+    "time": "60 m",
+    "fun": 1,
+    "copy-from": "cookbook_liverforkids"
+  },
+  {
+    "id": "cookbook_eatyrway",
+    "type": "BOOK",
+    "name": { "str": "Eat Your Way to a Fit Physique", "str_pl": "copies of Eat Your Way" },
+    "description": "This self-help book is as dubious as its title.  Worse, it is boring.  It does, however, contain a few basic cooking instructions.",
+    "weight": "800 g",
+    "volume": "1100 ml",
+    "required_level": 0,
+    "max_level": 1,
+    "time": "70 m",
+    "fun": -1,
+    "copy-from": "cookbook_liverforkids"
+  },
+  {
+    "id": "cookbook_foodfashions",
+    "type": "BOOK",
+    "name": { "str": "Food Fashions for Young Moderns", "str_pl": "copies of Food Fashions" },
+    "description": "A cookbook professing itself to be for those seeking \"daring and sophisticated flavors.\"  The book exhorts the reader to defy convention and break the rules, and then presents a number of useful rules and conventions for preparing food.",
```

Comment:
`\"daring and sophisticated flavors.\"`
Can quotation marks be used inside the description?

Reply:
Sure, if they're broken like that.